policies/attestation_task_bundle.rego

```diff
@@ -0,0 +1,36 @@
+package policies.attestation_task_bundle
+
+import data.lib
+
+allowed_bundles := ["quay.io/appstudio-utils:latest"]
+
+# METADATA
+# title: Task bundle was not used or is not defined
+# description: |-
+#   Check for existence of a task bundle. Enforcing this rule will
+#   fail the contract if the task is not called from a bundle.
```

Comment:
It would be useful to see which bundles are allowed in the docs. I like the approach that Simon took [here](https://github.com/hacbs-contract/ec-policies/blob/d71af3c03431fc3bbc39aed3c9a2d295efffe6d2/policies/step_image_registries.rego#L14-L21). Basically, define the allowed bundles as a metadata custom value, then read from it in the rule definition.

Reply:
Now that is cool.